server/src/KleeRunner.cpp

```diff
@@ -233,6 +242,18 @@ void KleeRunner::processBatchWithInteractive(const std::vector<tests::TestMethod
     if (!tests.isFilePresentedInArtifact) {
         return;
     }
+
+    for (const auto &method : testMethods) {
+        if (method.sourceFilePath != tests.sourceFilePath) {
+            std::stringstream message;
+            message << "While generating tests for source file: " << tests.sourceFilePath <<
+                       " tried to generate tests for method " << method.methodName << 
+                       " from another source file: " << method.sourceFilePath << '.' <<
+                       " This can cause invalid generation.\n";
+            LOG_S(WARNING) << message.str();
+        }
+    }
```

Comment:
Try to use StringUtils::stringFormat. It is the same as python format string. For [example](https://github.com/UnitTestBot/UTBotCpp/blob/6f9a1e0953672dc84b6642a4262318917dfbdf11/server/src/KleeRunner.cpp#L201).